libs/common/src/platform/services/environment.service.ts

```diff
@@ -86,6 +87,25 @@ export class EnvironmentService implements EnvironmentServiceAbstraction {
     return "https://vault.bitwarden.com";
   }
 
+  getCloudWebVaultUrl() {
+    if (this.cloudWebVaultUrl != null) {
+      return this.cloudWebVaultUrl;
+    }
+
+    return this.usUrls.webVault;
+  }
+
+  setCloudWebVaultUrl(region: Region) {
+    switch (region) {
+      case Region.EU:
+        this.cloudWebVaultUrl = this.euUrls.webVault;
+        break;
+      default:
```

Comment:
nit: it's nice to be explicit about what the default is:

```suggestion
      case Region.US:
      default:
```